requirements.txt

```diff
@@ -4,6 +4,7 @@
 #
 #    pip-compile --output-file=requirements.txt requirements.in
 #
+anglicize==0.0.3          # via -r requirements.in
```

Comment:
Is this a stable package ? The version number sounds a bit like it's still way in alpha.
I think it would be better to use https://pypi.org/project/Unidecode/ instead.

Reply:
Hmm, unidecode is GPLed, so perhaps not the best choice.

Reply:
This appears to be better: https://github.com/spanishdict/fold_to_ascii, also because it's based on what Lucene does.

Reply:
> The version number sounds a bit like it's still way in alpha.

If you'd like I can bump the version number of anglicize to 1.0.0 ;)

`fold_to_ascii` seems OK but I did run in to one issue.

```
Traceback (most recent call last):
  File "/code/tests/test_p3_templatetags.py", line 40, in test_spealers_templatetag
    data = sessions.speakers(conf.code)
  File "/code/p3/templatetags/sessions.py", line 214, in speakers
    speaker_list.sort(key=speaker_list_key)
  File "/code/p3/templatetags/sessions.py", line 70, in speaker_list_key
    return fold(name.strip().title())
  File "/usr/local/lib/python3.6/site-packages/fold_to_ascii/core.py", line 27, in fold
    if type(unicode_string) != unicode:
NameError: name 'unicode' is not defined
```

Admittedly this is probably because I'm trying to use it for something it wasn't designed for: preserve the unicode characters it can't convert.
My thinking was: 
 * we should do a best effort sorting for names written in a Latin based alphabet
 * preserve unicode for other names (that way we don't just group Chinese, Japanese, Cyrillic, etc as a single "other" group)

Reply:
Oh, sorry, didn't know that anglicize is your package :-)

The traceback looks like the code was written for Python 2. If it doesn't work, then let's use your package. That way we have more control over what it's doing. Sorry for not checking earlier.

Reply:
> Oh, sorry, didn't know that anglicize is your package :-)

No worries, and `fold_to_ascii` looks like a nice package. I'll make sure to keep it on my radar if I need something more serious.